public/not_configured.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use SimpleSAML\Configuration;
+use SimpleSAML\XHTML\Template;
+
+$globalConfig = Configuration::getInstance();
+$t = new Template($globalConfig, 'simpletotp:not_configured.twig');
+//$t->show();
```

Comment:
nit: let's remove this commented out code

Reply:
oops - I forgot to remove all of those.  Will work through them.